### PR TITLE
Update Japscan

### LINKED ISSUE #16898: [JapScan] Get Error during download : "Attendait 2 chaînes de recherche dans ZJS, a trouvé 0"
### Source information

JapScan 1.4.40

### Source language

French

### Steps to reproduce

Step to reproduce : 
1. Get scan from Japscan source (I my case is Mushoku Tensei)
2. Try to read chapter
3. An error appear after some seconds, and close the player

### Expected behavior

When i want to read chapter, the player should open and display chapter, without error

### Actual behavior

When I open a chapter, the player opens then after a few seconds it closes with an error

### Tachiyomi version

0.14.6

### Android version

Android 13

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.14.6](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by Rajh**
Japscan splitted tables, fast fix for it
Closes #16898

Checklist:

- [X] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [X] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [X] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [X] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [X] Have not changed source names
- [X] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

**Comment by Krylanc3lo** [comment]
Thanks a lot @Rajh !

File: src/fr/japscan/src/eu/kanade/tachiyomi/extension/fr/japscan/Japscan.kt [hunk 1]
```diff
@@ -281,15 +280,33 @@ class Japscan : ConfigurableSource, ParsedHttpSource() {
...
+        val rawShortStringLookupTables = shortDecodingStringsRe.findAll(zjs).map {
+            it.groupValues[1]
         }.toList()
 
-        if (stringLookupTables.size != 2) {
-            throw Exception("Attendait 2 chaînes de recherche dans ZJS, a trouvé ${stringLookupTables.size}")
+        if (rawShortStringLookupTables.size != 3) {
+            throw Exception("Attendait 3 chaînes de recherche dans ZJS, a trouvé ${rawShortStringLookupTables.size}")
         }
+        Log.d("japscan", "raw short lookup tables: $rawShortStringLookupTables")
```
Comment (alessandrojean) [inline src/fr/japscan/src/eu/kanade/tachiyomi/extension/fr/japscan/Japscan.kt:302]:
```suggestion
```

**Review by alessandrojean — CHANGES_REQUESTED** [review]

**Comment by ronycohen** [comment]
@Rajh

**Comment by ronycohen** [comment]
Thanks a lot @Rajh !

**Pull request merged**